schemas/resources/schemas/data-quality-summary/v1.0.0/metadata.json

```diff
@@ -3,9 +3,8 @@
   "Name":"CER-BDS_Terra",
   "Summary":"Summary",
   "MetadataSpecification": {
-    "Name": "DataQualitySummary",
+    "Name": "Data-Quality-Summary",
     "Version": "1.0.0",
-    "URL": "https://cdn.earthdata.nasa.gov/generics/dataqualitysummary/v1.0.0"
+    "URL": "https://cdn.earthdata.nasa.gov/generics/data-quality-summary/v1.0.0"
```

Comment:
I agree with dashes here, but I pushback slightly on the name.

Reply:
As in the name of generic itself, or that it should be something like "dataquality-summary" etc?

Reply:
Done, see comment above